src/main/java/lv/javaguru/java2/servises/RemovePlayerService.java

```
@@ -0,0 +1,27 @@
+package lv.javaguru.java2.servises;
+
+import lv.javaguru.java2.database.Database;
+import lv.javaguru.java2.domain.Player;
+
+import java.util.Optional;
+
+public class RemovePlayerService {
+
+    private Database database;
+
+    public RemovePlayerService(Database database) {
+        this.database = database;
+    }
+
+    public boolean removePlayerByNickname(String nickname) {
+        Optional<Player> foundPlayer = database.getByNickname(nickname);
+        if (foundPlayer.isPresent()) {
+            Player player = foundPlayer.get();
+            boolean isRemoved = database.remove(player);
+            return isRemoved;
+        } else {
+            return false;
```

Good practice is to throw an exception and handle it, instead of returning boolean